appkeeper/manifest.json

```diff
@@ -19,10 +19,13 @@
     "cloud",
     "processing"
   ],
-  "type": "crawler",
+  "type": "check",
   "is_public": false,
   "integration_id": "appkeeper",
   "assets": {
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
     "dashboards": {},
```

Comment:
in order to have the dashboard installed on the customer's account, you'll have to include the path to the dashboard here, ex:

`"dashboards" : "assets/dashboards/appkeeper_dashboard.json"`,